### ISSUE TITLE
Loss error

### ISSUE THREAD

**Issue by theFilipko** [hitachinsk/SAMed#27]
Hello, thank you for sharing the code.
Why do you calculate CE loss between logits and multi-label encoding? https://github.com/hitachinsk/SAMed/blob/e351840481f51d0c7dee50007f29d02197b63d99/trainer.py#L24
